docs/book/src/tasks/experimental-features/cluster-class/operate-cluster.md

```diff
@@ -42,3 +61,162 @@ kubeadmcontrolplane.controlplane.cluster.x-k8s.io/clusterclass-quickstart-XXXX
+```
+
+
+A new MachineDeployment can be added to the Cluster by adding a new MachineDeployment spec under `/spec/topology/workers/machineDeployments/`. To do so we can patch our Cluster with:
+```bash 
+kubectl  patch cluster capi-quickstart --type json --patch '[{"op": "add", "path": "/spec/topology/workers/machineDeployments/-",  "value": {"name": "second-deployment", "replicas": 1, "class": "default-worker"} }]'
+```
+This patch will make the below changes on the Cluster yaml:
+```diff
+   spec:
+     topology:
+       workers:
+         machineDeployments:
+         - class: default-worker
+           metadata: {}
+           replicas: 3
+           name: md-0
++        - class: default-worker
++          metadata: {}
++          replicas: 1
++          name: second-deployment
+```
+
+After a minute to scale the new MachineDeployment we get:
+```bash
+NAME                                      CLUSTER           REPLICAS   READY   UPDATED   UNAVAILABLE   PHASE     AGE   VERSION
+capi-quickstart-md-0-XXXX                 capi-quickstart   1          1       1         0             Running   39m   v1.23.3
+capi-quickstart-second-deployment-XXXX    capi-quickstart   1          1       1         0             Running   99s   v1.23.3
+```
+Our second deployment uses the same underlying MachineDeployment class `default-worker` as our initial deployment. In this case they will both have exactly the same underlying machine templates. In order to modify the templates MachineDeployments are based on take a look at [Changing a ClusterClass].
+
+A similar process described here - removing the machineDeployment from `cluster.spec.topology.workers.machineDeployments` - can be used to delete a running MachineDeployment from an active Cluster.
+
+## Scale a ControlPlane
+When using a managed topology scaling of ControlPlane Machines, where the Cluster is using a topology that includes ControlPlane MachineInfrastructure, should be done through the Cluster topology.
+
+This is done by changing the ControlPlane replicase field at `/spec/topology/controlPlane/replica` in the Cluster object. The command is:
+
+```bash 
+kubectl  patch cluster capi-quickstart --type json --patch '[{"op": "replace", "path": "/spec/topology/controlPlane/replicas",  "value": 1}]'
+```
+
+This patch will make the below changes on the Cluster yaml:
+```diff
+   spec:
+      topology:
+        controlPlane:
+          metadata: {}
++         replicas: 1
+-         replicas: 3
+```
+
+As well as scaling a ControlPlane, Cluster authors can edit the labels and annotations applied to a running ControlPlane using the Cluster topology as a single point of control.
+
+
+## Use variables
+A ClusterClass can use variables and patches in order to allow flexible customization of Clusters derived from a ClusterClass. Variable definition allows two or more Cluster topologies derived from the same ClusterClass to have different specs, with the differences defined in variables in the Cluster topology.
```

Comment:
```suggestion
A ClusterClass can use variables and patches in order to allow flexible customization of Clusters derived from a ClusterClass. Variable definition allows two or more Cluster topologies derived from the same ClusterClass to have different specs, with the differences controlled by variables in the Cluster topology.
```
Maybe slightly more correct, as the differences are defined in the patches in the ClusterClass.